app/scrapers/ask.py

```diff
@@ -25,7 +25,7 @@ def parseResponse(self, soup):
             title = div.div.a.text
             url = div.div.a['href']
             try:
-                p = div.find('p', {'class': 'PartialSearchResults-item-abstract'})
+                p = div.find('p', class_='PartialSearchResults-item-abstract')
```

Comment:
Why this change?

Reply:
PEP8 Compliance: This is equivalent functionality but requires fewer characters and PEP8 says that lines should 79 chars or less.

Reply:
The BeautifulSoup documentation tends to use the shorter from rather than creating and destroying a custom dict on every call.  https://www.crummy.com/software/BeautifulSoup/bs4/doc/#searching-by-css-class

Reply:
Ok, but I still find the object notation more readable :P 
That's why I was seeking other opinions @vaibhavsingh97

Reply:
I removed the PEP8 fix.  It is not worth the wait.